akka-docs/src/main/paradox/typed/cluster-membership.md

```diff
@@ -58,60 +58,74 @@ Note that the node might already have been shutdown when this event is published
+might still track the old instance as unreachable. In this case, however, it is clear that the old node is gone because
+the new instance will have the same address (host and port) as its old instance. In this case, the previous instance
+will be automatically marked as `down` and the new instance can rejoin the cluster without manual intervention.
+
+## Leader
+
+The purpose of the `leader` is to confirm state changes when convergence is reached. The `leader` can be determined
+by each node unambiguously after gossip convergence. Any node might be
+required to take the role of the `leader` depending on the current cluster composition.
+
+Without convergence, different nodes might have different views about which node is the leader.
+Therefore, most leader actions are only allowed if there is convergence to ensure that all nodes agree about the current
+state of the cluster and state changes are originated from a single node. Most regular state changes like changing
+a node from `joining` to `up` are of that kind.
+
+Other situations require that an action is taken even if convergence cannot be reached currently. Notably, convergence
+cannot be reached if one or more nodes in the cluster are currently unreachable as determined by the
+@ref[failure detector](cluster-concepts.md#failure-detector). In such a case, the cluster might be partitioned (a split brain
+scenario) and each partition might have its own view about which nodes are reachable and which are not. In this case,
+a node on each side of the partition might view itself as the leader of the reachable nodes. Any action that the leader
+performs in such a case must be designed in a way that all concurrent leaders would come to the same conclusion (which
+might be impossible in general and only feasible under additional constraints). The most important case of that kind is a split
+brain scenario where nodes need to be downed, either manually or automatically, to bring the cluster back to convergence.
+
+See the [Lightbend Split Brain Resolver](https://doc.akka.io/docs/akka-enhancements/current/split-brain-resolver.html)
+for an implementation of that.
+
+Another transition that is possible without convergence is marking members as `WeaklyUp` as described in the next section.
 
 <a id="weakly-up"></a>
 ## WeaklyUp Members
 
-If a node is `unreachable` then gossip convergence is not possible and therefore any
-`leader` actions are also not possible. However, we still might want new nodes to join
-the cluster in this scenario.
-
-`Joining` members will be promoted to `WeaklyUp` and become part of the cluster if
-convergence can't be reached. Once gossip convergence is reached, the leader will move `WeaklyUp`
-members to `Up`.
-
-This feature is enabled by default, but it can be disabled with configuration option:
-
-```
-akka.cluster.allow-weakly-up-members = off
-```
-
-You can subscribe to the `WeaklyUp` membership event to make use of the members that are
```

Comment:
This looks like a duplication left over from a merge? I didn't dig into what was supposed to be the ultimate result of the merge but mostly edited the text to make sense.